### PR TITLE
Fix error listing tags for rds db subnet group

### PR DESCRIPTION
**Pull request by szopen321**
Suggested solution to https://github.com/spulec/moto/issues/5479

### CODE REVIEW

File: moto/rds/models.py [hunk 1]
```diff
@@ -1485,8 +1492,12 @@ def authorize_security_group(self, security_group_name, cidr_ip):
         security_group.authorize_cidr(cidr_ip)
         return security_group
 
-    def create_subnet_group(self, subnet_name, description, subnets, tags):
-        subnet_group = SubnetGroup(subnet_name, description, subnets, tags)
+    def create_subnet_group(
+        self, subnet_name, description, subnets, tags, region, account_id
```
Comment (bblommers) [inline moto/rds/models.py:1496]: The region and account are already available in the backend, so we don't have to pass them explicitly.
`self.region_name` and `self.account_id` out the top of my head

**Review by bblommers — CHANGES_REQUESTED** [review]
Would you mind adding a test for this @szopen321? As it stands I don't immediately see what error it would fix

Edit: I just realized you made it clear what's wrong in the original issue. Verification that this value is now returned properly would still be nice though :)

**Comment by szopen321** [comment]
Oh, yes, my apologies! I've been meaning to mention aws docs to make the changes clearer.

This docs have information about arn missing:
https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_DescribeDBSubnetGroups.html

While below it is included:
https://docs.aws.amazon.com/cli/latest/reference/rds/describe-db-subnet-groups.html
https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_DBSubnetGroup.html

I used their feedback form in hopes they will complete the docs.

**Review by bblommers — APPROVED** [review]
Nice one - thanks for the fix @szopen321!

**Pull request merged**